lib/cli.js

```diff
@@ -1,15 +1,16 @@
-import yargs from 'yargs'
-import {log} from './utils/log'
-import {buildContext, getCommand, assertContext} from './utils/middlewares'
-import {version} from '../package.json'
+const yargs = require('yargs')
```

Comment:
Any reason we're not using the es6 import and export style?

Reply:
By using commonjs we can get rid of the babel transpilation step.. It's currently the only reason we have to transpile the code, all the other features are included from Node >= 8 anyways.

Reply:
Got it. 😢